.gitignore

```diff
@@ -4,4 +4,7 @@ logs/*
 node_modules/*
 
 helix-importer-ui
-.DS_Store
\ No newline at end of file
+.DS_Store
+.idea/
+.vscode/
+styles/styles-old.css
```

Comment:
When ready, we should delete the `styles/styles-old.css` file then delete this line before merging.

Reply:
removed